Scripts/Python/SISO/step_response_identification.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+import numpy as np
+from typing import Tuple
+import control
+from control.matlab import step
+import matplotlib.pyplot as plt
```

Comment:
Use the native function "control.step_response" instead of the MATLAB compatible version "control.matlab.step".

Reply:
Done